www/src/components/creative-layout.module.scss

```diff
@@ -9,14 +9,14 @@ $base-font-family: 'compasspromedium', 'Source Code Pro', monospace !default;
 $base-font-size: 17pt !default;
 $mobile-font-size: 17pt !default;
 $base-line-height: 1.2 !default;
-$container-width: 100% !default;
-$container-max-width: 860px !default;
+$container-width: calc(100% - 50px) !default;
+$container-max-width: 700px !default;
 
 // From dark
 :root {
   --base-color: #DBDBDB;
   --primary-color: #A2FC8F;
-  --border: dashed 1px rgba(219, 219, 219, 0.9);
+  --border: solid 1px rgba(219, 219, 219, 0.9);
   --selection-background: rgba(219, 219, 219, 0.99);
   --selection-text: #000;
   --background-color: #000;
```

Comment:
this should use spellsource purpleblack from here
`#21215c`

<img width="627" alt="Screen Shot 2020-06-29 at 12 27 21 PM" src="https://user-images.githubusercontent.com/2229300/86047581-f10c3a00-ba03-11ea-8975-aefc032679bc.png">